services/TotemAccountGateway.cs

```diff
@@ -0,0 +1,117 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Networking;
+using UnityEngine.Events;
+using System.Net;
+using utilities;
+
+
+namespace TotemServices
+{
+    public class TotemAccountGateway : MonoBehaviour
+    {
+        private readonly string _urlRoot = "https://account.totem.gdn/";
+        private readonly string _httpListenerUrl = "http://localhost:6700/auth/";
+
+        private readonly string _succesResponse = "<html><head><meta charset='utf8'></head><body>Login successful. You can close this page and return to the app</body></html>";
```

Comment:
Responses file maybe?

Reply:
Yea, i'll create a HTML file for the response